en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,453 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs and geospatial indexes with radius queries.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Redis Connector and access the Redis server, using Redis commands.
+
+The user sends the request for invoke to the created API to get IBM stock details. These REST call will get converted into SOAP calls and sent to the back-end service. The response will be sent to the client and in will use to create a `Redis list` to store IBM stock volume details inside the Redis server.
+If user need to retrieve list items (stock volume) from the created list need to invoke `getstockvolumelength` operation. Based on the known length user can list down  range of inserted list items. Finally user able to remove unwanted list items from the create list.
+
+> **Note**: In this scenario we need to set up the Redis Server in your local machine. Please refer the [Setting up the Redis Connector](redis-connector-configuration.md) documentation. Follow the steps listed under `Setting up the Redis server` section for setup the Redis server and `Set up the back-end service` section for setup the Stockquote service. 
+
+This example demonstrates how to use Redis connector to:
+
+1. Create a Redis list and add stock volume for the IBM inside the Redis server (A Redis list is location for storing stock volume details about IBM stocks).
```

Comment:
Is it only for IBM? As i understood it is for any Company.

Reply:
Our stock quote sample the request is routed to the IBM endpoint. 

1. https://docs.wso2.com/display/EI620/Sample+663%3A+Using+Multiple+Sequences
2. https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/examples/rest_api_examples/introduction-rest-api/

I have updated above as example doc first  point as like this. 

1. Create a Redis list and add stock volume details to the Redis server. (In this example a Redis list is use to storing IBM stocks volume details. Since the “symbol” that is sent in the request is “IBM”, the request is routed to the IBM endpoint. Once the response from the IBM endpoint is received, it is transformed according to the specified template and sent to the client. Then create a list and insert extracted details to the Redis list).

Reply:
Updated as "WSO2"